lib-ee/emqx_node_rebalance/src/emqx_node_rebalance_cli.erl

```diff
@@ -0,0 +1,279 @@
+cli(["node-status"]) ->
+    node_status(emqx_node_rebalance_status:local_status());
+cli(["status"]) ->
+    #{
+        evacuations := Evacuations,
+        rebalances := Rebalances
+    } = emqx_node_rebalance_status:global_status(),
+    lists:foreach(
+        fun({Node, Status}) ->
+            emqx_ctl:print(
+                "--------------------------------------------------------------------~n"
+            ),
+            emqx_ctl:print(
+                "Node ~p: evacuation~n~s",
+                [Node, emqx_node_rebalance_status:format_local_status(Status)]
+            )
+        end,
+        Evacuations
+    ),
+    lists:foreach(
+        fun({Node, Status}) ->
+            emqx_ctl:print(
+                "--------------------------------------------------------------------~n"
+            ),
+            emqx_ctl:print(
+                "Node ~p: rebalance coordinator~n~s",
+                [Node, emqx_node_rebalance_status:format_coordinator_status(Status)]
+            )
+        end,
+        Rebalances
+    );
+cli(["stop"]) ->
+    case emqx_node_rebalance_evacuation:status() of
+        {enabled, _} ->
+            ok = emqx_node_rebalance_evacuation:stop(),
+            emqx_ctl:print("Rebalance(evacuation) stopped~n"),
+            true;
+        disabled ->
+            case emqx_node_rebalance:status() of
+                {enabled, _} ->
+                    ok = emqx_node_rebalance:stop(),
+                    emqx_ctl:print("Rebalance stopped~n"),
+                    true;
+                disabled ->
+                    emqx_ctl:print("Rebalance is already disabled~n"),
+                    false
+            end
+    end;
+cli(_) ->
+    emqx_ctl:usage(
+        [
+            {
+                "rebalance start --evacuation \\\n"
+                "    [--redirect-to \"Host1:Port1 Host2:Port2 ...\"] \\\n"
+                "    [--conn-evict-rate CountPerSec] \\\n"
+                "    [--migrate-to \"node1@host1 node2@host2 ...\"] \\\n"
+                "    [--wait-takeover Secs] \\\n"
+                "    [--sess-evict-rate CountPerSec]",
+                "Start current node evacuation with optional server redirect to the specified servers"
+            },
+
+            {
+                "rebalance start \\\n"
+                "    [--nodes \"node1@host1 node2@host2\"] \\\n"
+                "    [--wait-health-check Secs] \\\n"
+                "    [--conn-evict-rate ConnPerSec] \\\n"
+                "    [--abs-conn-threshold Count] \\\n"
+                "    [--rel-conn-threshold Fraction] \\\n"
+                "    [--conn-evict-rate ConnPerSec] \\\n"
+                "    [--wait-takeover Secs] \\\n"
+                "    [--sess-evict-rate CountPerSec] \\\n"
+                "    [--abs-sess-threshold Count] \\\n"
+                "    [--rel-sess-threshold Fraction]",
+                "Start current node evacuation with optional server redirect to the specified servers"
```

Comment:
```suggestion
                "Start current node rebalance with optional server redirect to the specified servers"
```
?

Seems to be the same description as the first command.

Reply:
✅